go.mod

```diff
@@ -1,14 +1,10 @@
 module github.com/ethereum/go-ethereum
 
-go 1.15
+go 1.17
```

Comment:
Please run `go tidy`

Reply:
already updated the `go.mod` and `go.sum`